README.md

```diff
@@ -6,18 +6,23 @@ This repository contains a set of npm packages that deliver React utilities (Com
 | @bentley/property-grid-react       | [packages/property-grid](./packages/property-grid)             | [changelog](./packages/property-grid/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fproperty-grid-react.svg)](https://badge.fury.io/js/%40bentley%2Fproperty-grid-react)             |
 | @bentley/imodel-content-tree-react | [packages/imodel-content-tree](./packages/imodel-content-tree) | [changelog](./packages/imodel-content-tree/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fimodel-content-tree-react.svg)](https://badge.fury.io/js/%40bentley%2Fimodel-content-tree-react) |
 | @bentley/imodel-react-hooks        | [packages/imodel-react-hooks](./packages/imodel-react-hooks)   | [changelog](./packages/imodel-react-hooks/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fimodel-react-hooks.svg)](https://badge.fury.io/js/%40bentley%2Fimodel-react-hooks)               |
 | @bentley/geo-tools-react           | [packages/geo-tools](./packages/geo-tools)                     | [changelog](./packages/geo-tools/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fgeo-tools-react.svg)](https://badge.fury.io/js/%40bentley%2Fgeo-tools-react)                     |
 | @bentley/measure-tools-react       | [packages/measure-tools](./packages/measure-tools)             | [changelog](./packages/measure-tools/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fmeasure-tools-react.svg)](https://badge.fury.io/js/%40bentley%2Fmeasure-tools-react)             |
+| @bentley/breakdown-trees-react       | [packages/breakdown-trees](./packages/breakdown-trees)             | [changelog](./packages/breakdown-trees/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fbreakdown-trees-react.svg)](https://badge.fury.io/js/%40bentley%2Fbreakdown-trees-react)             |
+
+The following packages have been deprecated and will no longer be maintained:
+| Name                               | Folder                                                         | Changelog | Version                                                                                                                                                |
+| ---------------------------------- | -------------------------------------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| @bentley/imodel-select-react       | [packages/imodel-select](./packages/imodel-select/)            | [changelog](./packages/imodel-select/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fimodel-select-react.svg)](https://badge.fury.io/js/%40bentley%2Fimodel-select-react)             |
```

Comment:
Do we want to point to the admin component that should be used instead of this one?

Reply:
Yea i was wondering if i should do that here in the root readme or in the pkg readme, but then again i dont think with the way i have it setup it won't publish a new version of the pkg. 

I'll update to add a tab for replacements (pointing to admin-comp-react)

Reply:
Do we actually want to publish a new ver saying this pkg has been deprecated?